app/gui/src/model/suggestion_database/entry.rs

```diff
@@ -78,7 +91,7 @@ impl From<QualifiedName> for String {
 
 impl From<&QualifiedName> for String {
     fn from(name: &QualifiedName) -> Self {
-        name.into_iter().map(|s| s.deref()).join(ast::opr::predefined::ACCESS)
+        name.into_iter().map(|s| s.deref()).join(ACCESS)
```

Comment:
we prefer qualified name. The previous name was easier to read. I knew what is ACCESS, now I don't. Please revert it to the previosu form.